tracee-ebpf/tracee/co-re/vmlinux.h

```diff
@@ -66,7 +66,11 @@ enum {
 struct thread_info {
 	u32 status;
 };
-
+struct file_operations {
+	int (*iterate) (struct file *, struct dir_context *);
+	int (*iterate_shared) (struct file *, struct dir_context *);
+	
```

Comment:
This will add a new BTF TYPE kind and make BTFgen to stop working (it does not support BTF KINDs FUNC_PROTO and FUNC_PROTOTYPE). Needs https://github.com/kinvolk/btfgen/issues/17 to be solved first.